src/theme.ts

```diff
@@ -27,6 +27,29 @@ const theme = createMuiTheme({
         textTransform: "none",
       },
     },
+    MuiTabs: {
+      root: {
+        backgroundColor: "white",
+        color: "black",
+      },
+    },
+    MuiTab: {
+      root: {
+        minWidth: 0,
+        backgroundColor: "#E7E7E7",
+        borderRadius: "15px 15px 0px 0px",
+        border: "1px solid #C8C8C8",
+        fontWeight: 700,
+        textTransform: "none",
+        "@media (min-width: 0px)": {
```

Comment:
(outta curiosity) what does this media query do :o

Reply:
It's a CSS specificity trick! Without the media query, the default MUI styling for the min-width of a tab was overriding this, and we all know that nobody likes a fat 200 pixels tab 😄